dashboard/config/routes.rb

```diff
@@ -695,7 +695,7 @@
 
       post 'ml_models/save', to: 'ml_models#save'
       get 'ml_models/names', to: 'ml_models#user_ml_model_names'
-      get 'ml_models/:model_id', to: 'ml_models#get_trained_model'
+      get 'ml_models(/:model_id)', to: 'ml_models#get_trained_model'
```

Comment:
for clarity, we can organize these routes as a collection under the resource `:ml_models`: https://guides.rubyonrails.org/routing.html#adding-collection-routes